### ISSUE TITLE
Cocoapods not up to date?

### ISSUE THREAD

**Issue by dvluong** [jmenter/JAMSVGImage#45]
Is there a reason why when I download the current cocoapods, it does not have the latest codebase? I had to fork in order to get the latest one b/c the when I do pod install 'JAMSVGImage', the code to set the line width in the bezier path is missing which messes up my SVGs. For now I have forked it in order to work.

**Comment by jmenter** [comment]
Yeah, sorry this project has grow a little stale. Been busy with other thing.

**Comment by slxl** [comment]
@jmenter anyway, thanks for the work done!
meanwhile, do you have any plans to review/merge PRs and work on other features within the observable future?
